signup/signupController.js

```diff
@@ -142,33 +142,46 @@ define('signupController', [], function() {
 			});
 		};
 
-		$scope.intStudentChanged = function() {
+		$scope.updateAppointmentType = () => {
+			// TODO: This is hard-coded to true right now since all appointments are virtual, should eventually be replaced with a question
+			const virtualAppointmentRequest = true;
+			if (virtualAppointmentRequest && !$scope.sharedProperties.appointmentType.includes('virtual-')) {
+				$scope.sharedProperties.appointmentType = 'virtual-' + $scope.sharedProperties.appointmentType;
```

Comment:
Actually, I think the point of this change was to enable _any_ site to handle any appointment type--so you don't need a corresponding virtual site anymore. I think it was set up so that only sites that can handle the requested appointment type will be shown in the appointment picker

Reply:
Right, I meant any *appointment type* needs to have a corresponding virtual *type*

Reply:
Yah that's right--assuming the type allows for a corresponding virtual type. But I don't anticipate new types will really be added tbh